perfkitbenchmarker/benchmarks/cassandra_stress_benchmark.py

```diff
@@ -455,7 +267,9 @@ def CollectResults(benchmark_spec):
               'num_data_nodes': len(vm_dict[DATA_NODE]),
               'num_loader_nodes': len(vm_dict[LOADER_NODE]),
               'num_cassandra_stress_threads':
-              FLAGS.num_cassandra_stress_threads}
+              FLAGS.num_cassandra_stress_threads,
+              'scratch_disk_type': FLAGS.scratch_disk_type,
```

Comment:
I think scratch_disk_type and scratch_disk_size might be better to record at a higher level, since every benchmark involving disks will need these metadata.